redi/utils/GetEmrData.py

```diff
@@ -37,6 +39,7 @@ class EmrFileAccessDetails(object) :
     @see redi#_run()
     """
     def __init__(self,
+            emr_sftp_project_name,
             emr_download_file,
```

Comment:
I would rename this attribute to something like "download_list" to better reflect what we expect.